app/controllers/tasks_controller.rb

```diff
@@ -0,0 +1,68 @@
+require 'date'
+
+class TasksController < ApplicationController
+  def index
+    @tasks = Task.all
+  end
+
+  def show
+    task_id = params[:id].to_i
+    @task = Task.find(task_id)
+  end
+
+  def new
+    @task = Task.new
+  end
+
+  def create
+    @task = Task.new(action: params[:task][:action],
+                    description: params[:task][:description])
```

Comment:
This is a good way to do it, but it would be shorter and safer to use the `task_params` method you defined below. Just like how in the `update` method you used `task_params` on line 34, here, you could do:
```ruby
@task = Task.new(task_params)
```